server/routes.go

```diff
@@ -1640,3 +1642,23 @@ func handleScheduleError(c *gin.Context, name string, err error) {
 		c.JSON(http.StatusInternalServerError, gin.H{"error": err.Error()})
 	}
 }
+
+var thinkTagRegexp = regexp.MustCompile(`<think>(?s).*?</think>(\n)*`)
+
+func filterThinkTags(msgs []api.Message, theModel *Model) []api.Message {
```

Comment:
```suggestion
func filterThinkTags(msgs []api.Message, m *Model) []api.Message {
```

`m` might be acceptable here too given it's used early on in the function